File: dataset/output/OPENAI_gpt-4o/Task145_OPENAI_gpt-4o.java
import java.util.*;

public class MaxSubarraySum {
    public static void main(String[] args) {
        int[] nums = Arrays.stream(args).mapToInt(Integer::parseInt).toArray();
        System.out.println("Java: " + maxSubArray(nums));
    }

    public static int maxSubArray(int[] nums) {
        int maxSum = nums[0];
        int currentSum = nums[0];
        for (int i = 1; i < nums.length; i++) {
            currentSum = Math.max(nums[i], currentSum + nums[i]);
            maxSum = Math.max(maxSum, currentSum);
        }
        return maxSum;
    }
}

File: dataset/output/OPENAI_gpt-4o/Task145_OPENAI_gpt-4o.c
#include <stdio.h>
#include <stdlib.h>

int maxSubArray(int *nums, int length) {
    int maxSum = nums[0];
    int currentSum = nums[0];
    for (int i = 1; i < length; i++) {
        currentSum = (nums[i] > currentSum + nums[i]) ? nums[i] : currentSum + nums[i];
        maxSum = (maxSum > currentSum) ? maxSum : currentSum;
    }
    return maxSum;
}

int main(int argc, char *argv[]) {
    int nums[argc - 1];
    for (int i = 1; i < argc; i++) {
        nums[i - 1] = atoi(argv[i]);
    }
    printf("C: %d
", maxSubArray(nums, argc - 1));
    return 0;
}